test/tf/hcloud/versions.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    hcloud = {
+      source = "terraform-providers/hcloud"
```

Comment:
Could you pin the provider to a version requirement here? We've had issues in the past of providers updating in the background to latest and failing tests when older resource arguments/attributes get deprecated.